upload_components/upload.sh

```diff
@@ -1,5 +1,21 @@
 #!/bin/bash
 
+function check_if_exists {
+    EXIT_CODE=$?
```

Comment:
An optional recommendation, what about adding an `--allow-existing` flag to `create-remote-component`, `create-remote-namespace` commands? If the flag is given and the component/namespace exists, then the exit code will be 0. I think this is a bit more robust than relying on error code values.

Reply:
yes. I agree. I'll refactor a bit on the `idf-component-manager` side. The current "run as a module" way does not support different options for different operation.